.github/workflows/ci.yml

```
@@ -32,9 +32,18 @@ jobs:
             echo "Working tree dirty at end of job"
             exit 1
           fi
+  pr-linter:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      # v5.2.0
+      - name: Convetional Commit Validation PR Title
+        uses: amannn/action-semantic-pull-request@c3cd5d1ea3580753008872425915e343e351ab54
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   scripts:
     runs-on: ubuntu-20.04
-    needs: setup
+    needs: pr-linter
```

It might be better to leave this as-is, and add `pr-linter` as a requirement of the `all-jobs-pass` step. That way it runs in parallel rather than serially

---

This is now even more urgent since setup was removed from pr-linter. Switching this back is required to prevent this job from being broken


```suggestion
    needs: setup
```

---

Should be addressed and thanks for the feedback.

---

I'm not sure what you mean. This is not addressed, and as a result this job is no longer using the cached dependencies from the setup job.

---

In a better spot now. Let me know what you think.